src/app/frontend/login/component.ts

```diff
@@ -65,6 +65,11 @@ export class LoginComponent implements OnInit {
       .get<EnabledAuthenticationModes>('api/v1/login/modes')
       .subscribe((enabledModes: EnabledAuthenticationModes) => {
         this.enabledAuthenticationModes_ = enabledModes.modes;
+        Object.entries(LoginModes).forEach(([_, mode]) => {
+          if (mode === enabledModes.modes[0]) {
```

Comment:
Is there ever a condition or possibility of `enabledModes.modes` to have no entries? Always worry about hard coding an array index.

Reply:
How about simply switching order in the template, so that kubeconfig option is always last? This way enabled mode will be first anyway.

Reply:
> How about simply switching order in the template, so that kubeconfig option is always last? This way enabled mode will be first anyway.

We need to set selectedAuthenticationMode to one of enabeldModes explicitly. Otherwise, if hard coded default selectedAuthenticationMode is not included in enabled mode, i.e. enabled mode was set as only `basic` but hard coded default is `token`, selection for token does not exist but input field is shown as `token`.